index.js

```diff
@@ -1,20 +1,252 @@
+  ["slingshot"],
+  true,
+  ["clearing", "bridge"]
+);
+
+let bridge = new Location(
+  "bridge",
+  "a rickety looking bridge crosses a deep chasm. There is another fairy like creature standing at the start of the bridge.",
+  [],
+  true,
+  ["stumpRoom"]
+);
+
+let meadow = new Location(
+  "meadow",
+  "a grassy meadow with a large pond and a tree at it's edge. Above the pond hanging from the tree is a bullseye.",
+  [],
+  false,
+  ["clearing", "pasture"]
+);
+
+let pasture = new Location(
+  "pasture",
+  "a wide open pasture with a castle in the distance",
+  [],
+  false,
+  ["meadow", "castle"]
+);
+
+//Lookup table for locations, not working with extra names (stump, woods, etc)
+let locLookup = {
+  home: home,
+  balcony: balcony,
+  clearing: clearing,
+  woods: clearing,
+  stump: stumpRoom,
+  stumpRoom: stumpRoom,
+  bridge: bridge,
+  meadow: meadow,
+  pasture: pasture,
+  //castle: castle,
+};
+
+start();
+//A function to start the game
 async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
-There is a door here. A keypad sits on the handle.
-On the door is a handwritten sign.`;
-  let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
-  process.exit();
+  const welcomeMessage = `You awake from a dream in which you see a young child fleeing a large castle on horseback. She was escorted by a guardian... was she someone important...? The man she was fleeing was strange and had a truly evil look about him. He had olive green skin, glowing amber eyes and deep red hair. But enough about them... this is about YOU. So you wake up from this dream and you appear to be in a tree. Not in a tree-house, inside of a tree! But it's homey... there's a bed, some tables and chairs and other homey stuff. There's also a door leading to a balcony...`;
+  console.log(welcomeMessage);
 }
+
+//The main funtion to play the game
+async function play() {
+  let userAction = await ask(`>_`); //prompt that will accept user inputs
+  let inputArray = userAction.toLowerCase().split(" ");
+  let action = inputArray[0];
+  let target = inputArray[1];
+  if (action === "take") {
+    console.log(lookupTable[target].take());
+  } else if (action === "drop") {
+    console.log(lookupTable[target].drop());
+  } else if (action === "i" || action === "inventory" || action === "I") {
+    console.log(`You have:\n ${playerInventory.toString("\n")}`); //the player can check their inventory and it will be printed back to them
+  } else if (action === "use") {
+    console.log(lookupTable[target].use());
+  } else if (action === "read") {
+    console.log(lookupTable[target].read()); //
+  } else if (action === "move" || action === "go") {
+    console.log(locLookup[currentLocation].move(target));
```

Comment:
You should be using the `return` keyword, but you can get away with just calling the functions (which should be removed from the item constructor and on the global level) `if(action ==="take"){ take() }` or `if(action ==="take"){ return take() }`